cl/opinion_page/views.py

```diff
@@ -293,8 +292,13 @@ def view_opinion(request, pk, _):
             'caller': 'view_opinion',
         }
         citing_clusters = conn.raw_query(**q).execute()
+
+        # Get recommendations with MoreLikeThis query
+        mlt_query = conn.query(id=pk).mlt('text').field_limit(fields=['id', 'caseName', 'absolute_url'])
```

Comment:
1. Is this limited to the top five items somehow? Sorry if I'm not seeing it....

1. I guess we won't know until later, when we can do some performance checks, but it seems like we should be caching this. Even just doing it in the DB layer might be good. We have this configured here: https://github.com/freelawproject/courtlistener/blob/adbed32211691d17ee357e7ff774d1aea9810dd5/cl/settings/10-public.py#L214-L217

    I wonder how often this would change. Probably could have really long cache times, I'd bet?